docs/source/schema/directives.md

```diff
@@ -110,13 +110,11 @@ If you want to define a custom schema directive to _transform_ your executable s
 > Before you use directives in a federated graph, make sure to consider the following:
 >
 > - Custom directives are _not_ included in your graph's composed supergraph schema. The composition process strips all subgraph directives. Only a given subgraph is aware of its own directives.
```

Comment:
This is not necessarily true. I think this is a piece that may need retooling as well since this reads like "all directives are removed" when they are not; executable directives (client-side directives) are available in the supergraph if all subgraphs agree on the definition, and we can now also make server-side directives via `@composeDirective` (https://www.apollographql.com/docs/federation/federated-types/federated-directives/#composedirective).

Reply:
Thanks for pointing that out. The [type system directive section](https://www.apollographql.com/docs/federation/federated-types/composition/#type-system-directives) says that type system directives "are not composed into the supergraph schema, but they can still provide information to the router via the [@composeDirective](https://www.apollographql.com/docs/federation/federated-types/federated-directives#composedirective) directive."

Would you happen to know more about what "providing information to the router" means? Is it functionally making the directive available on the supergraph schema?

Reply:
> Would you happen to know more about what "providing information to the router" means? Is it functionally making the directive available on the supergraph schema?

Yes, it's just that. Before `@composeDirective`, the custom directives were simply stripped (iirc) from the schema.